test/rules/prefer-conditional/test.ts.lint

```diff
@@ -0,0 +1,37 @@
+let x;
+if (true) {
+~~ [X]
+    x = 1;
+} else {
+    x = 2;
+}
+
+if (true)
+~~ [X]
+    x = "TRUE";
```

Comment:
Yeah, that's a good example. In that case I personally would prefer to keep the if / else blocks; I generally only want to use ternarys when each RHS expression is a single line. If this is going to be a core rule, I think it should be more conservative than it is now -- can you make that change?